packages/flutter/lib/src/widgets/scroll_position.dart

```diff
@@ -12,19 +12,29 @@ import 'package:flutter/scheduler.dart';
 import 'basic.dart';
 import 'framework.dart';
 import 'gesture_detector.dart';
+import 'scroll_activity.dart';
+import 'scroll_context.dart';
 import 'scroll_metrics.dart';
+import 'scroll_notification.dart';
 import 'scroll_physics.dart';
 
 abstract class ScrollPosition extends ViewportOffset with ScrollMetrics {
```

Comment:
add a comment here so that when we add dartdocs to ScrollPosition we remember to add a "Subclassing ScrollPosition" section that talks about:
* responsibilities when implementing `absorb()` (that you may have to call goIdle, that you may have to update the activity's idea of what the delegate is, etc).
* something about userScrollDirection